netutil/interface_test.go

```diff
@@ -195,7 +195,10 @@ func TestNextBundleInterface(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.desc, func(t *testing.T) {
-			got, err := nextBundleInterface(test.vendor, test.intfs)
+			val := (&telemetry.QualifiedDevice{}).SetVal(&telemetry.Device{
```

Comment:
please change this to:
val := &telemetry.QualifiedDevice{}
if ints != nil {
  val.SetVal(&telemetry.Device{Interface: test.intfs})
}
to properly test the path where there is no value present

same below

Reply:
thanks, done